src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -11,6 +11,8 @@
       location,
       timezone,
       description,
+      staffInformationTitle: staffInformationTitle.display,
+      staffInformationOfficeLocation: staffInformationOfficeLocation.display,
```

Comment:
This could be confusing in future, but there's only one instance of it. Just how easy is it to update the mutation variable name to `staffInformationTitleDisplay` and `staffInformationOfficeLocationDisplay` on the client side?

Reply:
I'll actually also update the back-end. You're right here.